src/test/java/org/pesmypetcare/webservice/thirdpartyservices/adapters/firestore/FirestoreCollectionAdapterTest.java

```diff
@@ -113,34 +139,35 @@ public void getDocumentsWhereFieldEqualToValue() {
             given(collectionReference.whereEqualTo(same(field), same(value))).willReturn(query);
             Query result = collectionAdapter.getDocumentsWhereEqualTo(collectionPath, field, value);
             assertEquals(query, result,
-                "Should return all documents from " + collectionPath + " where " + field + " is equals to " + value);
+                         "Should return all documents from " + collectionPath + " where " + field + " is equals to "
+                             + value);
         }
 
         @Test
         public void getDocumentsWhereFieldFromFieldPathEqualToValue() {
             given(collectionReference.whereEqualTo(same(fieldPath), same(value))).willReturn(query);
             Query result = collectionAdapter.getDocumentsWhereEqualTo(collectionPath, fieldPath, value);
             assertEquals(query, result,
-                "Should return all documents from " + collectionPath + " where " + field + " in " + fieldPath
-                    + " is equals to " + value);
+                         "Should return all documents from " + collectionPath + " where " + field + " in " + fieldPath
+                             + " is equals to " + value);
         }
 
         @Test
         public void getDocumentsWhereArrayContainsValue() {
             given(collectionReference.whereArrayContains(same(array), same(value))).willReturn(query);
             Query result = collectionAdapter.getDocumentsWhereArrayContains(collectionPath, array, value);
             assertEquals(query, result,
-                "Should return all documents from " + collectionPath + " where the array " + array + " contains "
-                    + value);
+                         "Should return all documents from " + collectionPath + " where the array " + array
+                             + " contains " + value);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String " contains " appears 3 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5465948)